specification/storage/resource-manager/Microsoft.Storage/2017-06-01/storage.json

```diff
@@ -700,6 +731,34 @@
       ],
       "description": "The parameters used to check the availabity of the storage account name."
     },
+    "SKUCapability": {
+      "properties": {
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of capability, The capability information in the specified sku, including file encryption, network acls, change notification, etc."
+        },
+        "value": {
```

Comment:
Got it. Note that for Python [it would have work ok even with the string](https://github.com/Azure/msrest-for-python/blob/master/msrest/serialization.py#L1185).